tests/Replication2/ReplicatedLog/CalcCommitIndexTest.cpp

```diff
@@ -54,21 +54,34 @@ struct CalcCommitIndexTest : ::testing::Test {
     return buf.str();
   }
 
+  auto createDefaultTermIndexPair(decltype(LogIndex::value) value)
+      -> TermIndexPair {
+    return {LogTerm{1}, LogIndex{value}};
+  }
+
```

Comment:
Either put this function (and while you're at it, the other helper functions as well) in an anonymous namespace. Otherwise they are publicly visible in the global namespace. This might cause strange bugs, if it results in an ODR violation.